2002-07-24  Hrvoje Niksic  <hniksic@xemacs.org>

	* recur.c (retrieve_tree): Check whether downloaded_html_set is
	non-NULL before using it.

Index: src/recur.c
===================================================================
RCS file: /pack/anoncvs/wget/src/recur.c,v
retrieving revision 1.51
diff -u -r1.51 recur.c
--- src/recur.c	2002/05/18 02:16:24	1.51
+++ src/recur.c	2002/07/24 21:08:54
@@ -254,7 +254,8 @@
 	  DEBUGP (("Already downloaded \"%s\", reusing it from \"%s\".\n",
 		   url, file));
 
-	  if (string_set_contains (downloaded_html_set, file))
+	  if (downloaded_html_set
+	      && string_set_contains (downloaded_html_set, file))
 	    descend = 1;
 	}
       else
